dbms/src/Common/MemoryTracker.h

```diff
@@ -157,6 +164,16 @@ extern thread_local MemoryTracker * current_memory_tracker;
 extern std::shared_ptr<MemoryTracker> root_of_non_query_mem_trackers;
 extern std::shared_ptr<MemoryTracker> root_of_query_mem_trackers;
 
+// Initialize in `initStorageMemoryTracker`.
+// If a memory tracker of storage tasks is driven by query, it should inherit `sub_root_of_query_storage_task_mem_trackers`.
+// root_of_query_mem_trackers
+//             |-- sub_root_of_query_storage_task_mem_trackers
+//                                    |-- rn_fetch_pages_mem_tracker
+extern std::shared_ptr<MemoryTracker> sub_root_of_query_storage_task_mem_trackers;
```

Comment:
can we print usage of `sub_root_of_query_storage_task_mem_trackers` and `rn_fetch_pages_mem_tracker` along with the existed err log, when out of quota occur in `memtracker::alloc(size)`